### PR TITLE
Azure - Container Host Docs Cleanup

### PR DESCRIPTION
**Pull request by axis7818**
#4651 

Introduces the tutorial for deploying the Container Host to kubernetes and cleanup some recent changes to variable names.

This also adds some extra environment variables to the ACI ARM template for logging and metrics output.

### CODE REVIEW

File: docs/source/azure/configuration/acitutorial.rst [hunk 1]
```diff
@@ -34,6 +34,7 @@ the custodian output files of running policies.
 
     account_key=$(az storage account keys list --account-name c7nstorage --query "[0].value" --output tsv)
     az storage container create --account-name c7nstorage --account-key $account_key --name c7n-aci-policies
+    az storage container create --account-name c7nstorage --account-key $account_key --name c7n-aci-logs
```
Comment (logachev) [inline docs/source/azure/configuration/acitutorial.rst]: Add some text comment about the container for logs?

File: docs/source/azure/configuration/acitutorial.rst [hunk 2]
```diff
@@ -82,16 +98,25 @@ Create a parameters file called ``c7n-aci-tutorial.json``
                 "value": "c7n-aci"
             },
             "azure_subscription_id": {
-                "value": "<Target Subscription Id>"
+                "value": "<Target Subscription ID>"
             },
-            "azure_event_queue_name": {
+            "azure_container_queue_name": {
```
Comment (logachev) [inline docs/source/azure/configuration/acitutorial.rst]: nit: would be great to somehow separate c7n Container Host & Azure Storage Container.. Feel a little confusing

File: docs/source/azure/configuration/helmtutorial.rst [hunk 3]
```diff
@@ -4,4 +4,194 @@ Tutorial - Helm Deployment
...
+
+We will also create a blob container. One will host the uploaded policies, and the other will store
+the custodian output files of running policies.
+
+.. code-block:: bash
+
+    az storage account create --resource-group c7n-helm-tutorial --name c7nstorage
+
+    account_key=$(az storage account keys list --account-name c7nstorage --query "[0].value" --output tsv)
+    az storage container create --account-name c7nstorage --account-key $account_key --name c7n-helm-policies
```
Comment (logachev) [inline docs/source/azure/configuration/helmtutorial.rst]: nit: do we really need account key here?

File: docs/source/azure/configuration/acitutorial.rst [hunk 4]
```diff
@@ -63,11 +64,26 @@ storage account created above.
     az role assignment create --assignee <Managed Identity Client Id> \
         --role "Storage Queue Data Contributor" --scope <Storage Account Resource Id>
 
-4. Create the ACI Container Host
+4. Create an Application Insights Instance
+------------------------------------------
+
+We will point the container host to this Application Insights to gather logs and monitor the host
```
Comment (jomalsan) [inline docs/source/azure/configuration/acitutorial.rst]: "We will create an Application Insights instance to gather the logs and telemetry generated by the running container host"

File: docs/source/azure/configuration/acitutorial.rst [hunk 5]
```diff
@@ -63,11 +64,26 @@ storage account created above.
     az role assignment create --assignee <Managed Identity Client Id> \
         --role "Storage Queue Data Contributor" --scope <Storage Account Resource Id>
```
[the remark this answers is not in the document]
Comment (jomalsan) [reply docs/source/azure/configuration/acitutorial.rst]: For the Storage Account Resource Id I would mention either here or in step 2 that it is the id returned in the JSON response from creating a new storage account

File: docs/source/azure/configuration/acitutorial.rst (hunk 2 above)
Comment (axis7818) [reply docs/source/azure/configuration/acitutorial.rst]: Do you mean semantically separate them? I agree that it is a bit confusing. I could specify "blob container" vs "container host".

File: docs/source/azure/configuration/helmtutorial.rst (hunk 3 above)
Comment (axis7818) [reply docs/source/azure/configuration/helmtutorial.rst]: yeah, you do when using azure cli to create blob containers at least.

File: docs/source/azure/configuration/acitutorial.rst [hunk 6]
```diff
@@ -112,7 +137,7 @@ with the following command:
```
Comment (jomalsan) [inline docs/source/azure/configuration/acitutorial.rst]: Line 131: I think the @ sign is a typo in `--parameters @c7n-aci-tutorial.json`

File: docs/source/azure/configuration/helmtutorial.rst [hunk 7]
```diff
@@ -4,4 +4,194 @@ Tutorial - Helm Deployment
...
+
+    az group create --name c7n-helm-tutorial --location westus2
+
+2. Create a Storage Account
+---------------------------
+
+This storage account will hold all of the blobs and queues used by the container host. You could 
+also use an existing resource here or multiple storage accounts.
+
+We will also create a blob container. One will host the uploaded policies, and the other will store
```
Comment (jomalsan) [inline docs/source/azure/configuration/helmtutorial.rst]: nit: "We will also create two blob containers."

File: docs/source/azure/configuration/helmtutorial.rst [hunk 8]
```diff
@@ -4,4 +4,194 @@ Tutorial - Helm Deployment
...
+    az role assignment create --assignee <Service Principal Client Id> \
+        --role "Storage Blob Data Contributor" --scope <Storage Account Resource Id>
+
+    az role assignment create --assignee <Service Principal Client Id> \
+        --role "Storage Queue Data Contributor" --scope <Storage Account Resource Id>
+
+4. Create an Application Insights Instance
+------------------------------------------
+
+We will point the container host to this Application Insights to gather logs and monitor the host
```
Comment (jomalsan) [inline docs/source/azure/configuration/helmtutorial.rst]: "We will create an Application Insights instance to gather the logs and telemetry generated by the running container host"

File: docs/source/azure/configuration/helmtutorial.rst [hunk 9]
```diff
@@ -4,4 +4,194 @@ Tutorial - Helm Deployment
...
+5. Create an AKS Cluster and Install Tiller
+-------------------------------------------
+
+This AKS cluster will be used to host the pods running the Container Host. Tiller will also need to 
+be installed with the proper service account.
+
+.. code-block:: bash
+
+    az aks create --resource-group c7n-helm-tutorial --name c7n-helm \
+        --service-principal <Service Principal ID> --client-secret <Client Secret>
```
Comment (jomalsan) [inline docs/source/azure/configuration/helmtutorial.rst]: I would explicitly specify the node-count and the node-vm-size so people don't accidentally provision a few hundred dollars a month cluster

File: docs/source/azure/configuration/acitutorial.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (axis7818) [reply docs/source/azure/configuration/acitutorial.rst]: yeah, good idea

File: docs/source/azure/configuration/acitutorial.rst (hunk 6 above)
Comment (axis7818) [reply docs/source/azure/configuration/acitutorial.rst]: I think the `@` sign indicates to the [azure cli](https://docs.microsoft.com/en-us/cli/azure/group/deployment?view=azure-cli-latest#az-group-deployment-create) that it is a path to a parameters file.

**Pull request merged**